app/views/organization_reports/pages/_campaign.html.erb

```diff
@@ -4,27 +4,27 @@
     <p class="lead"><%= report.start_date.to_s("bdy") %> to <%= report.end_date.to_s("bdy") %></p>
     <div class="flex-container" style="margin-top: .5in;">
       <div class="flex-item">
-        <h4><%= number_to_currency summary.gross_revenue_cents / 100 %></h4>
+        <div class="h4"><%= number_to_currency summary.gross_revenue_cents / 100 %></div>
```

Comment:
why are these divs? Do you just need them to be block elements?

Reply:
They are block elements in the UI. I could have used span, but div made more sense in this case. I couldn't use H4 b/c docraptor uses those to create outlines.